docs/content/concepts/ops-jobs-graphs/jobs.mdx

```diff
@@ -1,32 +1,33 @@
 ---
 title: Jobs | Dagster
-description: A job is an executable graph of ops, with optional resources and configuration.
+description: Jobs are the main unit of execution and monitoring in Dagster.
 ---
 
 # Jobs
 
-Jobs are the main unit of execution and monitoring in Dagster. The core of a job is a [graph](/concepts/ops-jobs-graphs/graphs) of [ops](/concepts/ops-jobs-graphs/ops) connected via data dependencies.
+Jobs are the main unit of execution and monitoring in Dagster.
 
-Ops are linked together by defining the dependencies between their inputs and outputs. An important difference between Dagster and other workflow systems is that, in Dagster, op dependencies are expressed as data dependencies, not just execution dependencies.
+A job does one of two things:
```

Comment:
Did you take out everything about dependencies because asset dependencies are modeled differently than those for ops?